trashtalk/settings.py

```diff
@@ -21,6 +21,7 @@ class Config(object):
     TESTING = False
     SECRET_KEY = os.getenv('SECRET_KEY', 'Not set')
     SESSION_COOKIE_NAME = 'trashtalk_session'
+    SEE_ADMIN_USER = os.getenv('SCF_ADMIN_USER')
```

Comment:
Duplicate. It's already defined below so lets delete this line.

what's the difference between this admin and the one in `seeclickfix.py ADMIN_USER`?

Reply:
Redundant code is redundant. This line is now gone.